src/minishell/minishell.c

```diff
@@ -45,7 +45,8 @@ int32_t	minishell(char **envp)
 		sheldon.status = message_system_call_error("isatty");
 		rl_outstream = stdin;
 	}
-	while (minishell_loop(&sheldon))
+	sheldon.stop = false;
+	while (minishell_loop(&sheldon) && !sheldon.stop)
```

Comment:
beetje mierenneukerig, maar meer leesbaar is: sheldon.stop == true, hoef je niet persee te veranderen hoor

Reply:
Deze zou ik in het geval van die andere cases maar gewoon weghalen, er is toch nog wat werk aan de winkel